templates/patterns/offcanvas/pattern-offcanvas.html.twig

```diff
@@ -5,7 +5,11 @@
  */
 #}
 {% include '@oe-bcl/bcl-offcanvas/offcanvas.html.twig' with {
-  'title': title,
+  'title': {
```

Comment:
this revert, move the extra stuff to templates/overrides/search/block--facets-form.html.twig in whitelabel